code/modules/mob/living/superior_animal/giant_spider/giant_spider.dm

```diff
@@ -58,23 +57,22 @@
 	get_light_and_color(parent)
 	..()
 
-/mob/living/superior_animal/giant_spider/AttackingTarget()
+/mob/living/superior_animal/giant_spider/attemptAttackOnTarget()
 	var/target = ..()
+	if (target)
+		playsound(src, 'sound/weapons/spiderlunge.ogg', 30, 1, -3)
```

Comment:
stick a prob call around this, because hearing that sound every time is going to get pretty stale and lose the novelty